Rdmp.Core/Databases/CatalogueDatabase/up/079_AllowMultipleCatalogueLoadMetadatas.sql

```diff
@@ -0,0 +1,39 @@
+﻿--Version: 8.1.5
+--Description: Adds linking table to allow for multiple load metadatas per catalogue
+if not exists (select 1 from INFORMATION_SCHEMA.COLUMNS where TABLE_NAME='LoadMetadataCatalogueLinkage')
+BEGIN
+CREATE TABLE [dbo].LoadMetadataCatalogueLinkage(
+	[ID] [int] IDENTITY(1,1) NOT NULL,
+	[LoadMetadataID] [int] NOT NULL,
+	[CatalogueID] [int] NOT NULL,
+	FOREIGN KEY ([LoadMetadataID]) REFERENCES [dbo].[LoadMetadata](ID) ON DELETE CASCADE,
+	FOREIGN KEY ([CatalogueID]) REFERENCES [dbo].[Catalogue](ID) ON DELETE CASCADE,
+	CONSTRAINT [PK_LoadMetadataCatalogueLinkage] PRIMARY KEY CLUSTERED
+(
+	[ID] ASC
+)WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
+) ON [PRIMARY]
+END
+
+Declare @MigrateLinkageSQL varchar(max) = '
+    insert into [dbo].[LoadMetadataCatalogueLinkage](CatalogueID, LoadMetadataID)
+	select ID as CatalogueID, LoadMetadata_ID
+	from [dbo].[Catalogue]
+	where LoadMetadata_ID is not null'
+if exists (select 1 from INFORMATION_SCHEMA.COLUMNS where TABLE_NAME='Catalogue' and COLUMN_NAME='LoadMetadata_ID')
+BEGIN
+EXEC(@MigrateLinkageSQL)
```

Comment:
Is this SP reused, or could it just be called directly?
No indexing on the LoadMetadataID and CatalogueID columns?

Reply:
It's not used, have migrated it to just be called directly and added indexes for both IDs